### PR TITLE
Issue122 support

### PR DESCRIPTION
**Pull request by venturewise**
Test URLs:
- Before: https://main--merative2--hlxsites.hlx.page/events/2023-zelta-user-conference
- After: https://issue122-support--merative2--hlxsites.hlx.page/events/2023-zelta-user-conference

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -678,33 +677,33 @@ button {
...
 .section > div:first-child > *:first-child > *:first-child { /* no top margin on whatever comes next + 1 inner */
   margin-top: 0;
   padding-top: 0;
-  }
+}
 
 .button-container > a.tertiary:hover span.icon svg * {
-  stroke: url("#gradient");
-  }
+  stroke: var(--primary-red);
```
Comment (arumsey) [inline styles/styles.css:697]: Is this according to Figma design? Can https://github.com/hlxsites/merative/issues/164 be closed too then?

File: styles/styles.css [hunk 2]
```diff
@@ -665,7 +664,7 @@ button {
   -webkit-background-clip: text;
   /* stylelint-disable-line */
   -webkit-text-fill-color: transparent;
-  color: transparent;
+  color: var(--gradient-left-right);
```
Comment (arumsey) [inline styles/styles.css:667]: Why was this needed? Cross browser support?

File: styles/styles.css [hunk 3]
```diff
@@ -378,19 +377,15 @@ main img {
 /* Image variations */
 .leadership-content img {
     border-radius: 15px;
-    }
-
-main .quarter-circle img {
-    clip-path: circle(100% at 100% 100%);
```
Comment (arumsey) [inline styles/styles.css:384]: How do images get the quarter circle clip path applied now? Or is up to the images to already be cropped?

File: styles/styles.css (hunk 1 above)
Comment (venturewise) [reply styles/styles.css:697]: There isn't actually a spec for hover that includes the arrow, so I made an executive decision to use the last color of the gradient that is on the text link. I think it blends in/has continuity so I'm ready to call this ticket "complete" if you are.
image

File: styles/styles.css (hunk 2 above)
Comment (venturewise) [reply styles/styles.css:667]: ooh, no I need to put that back to transparent. Thanks!

File: styles/styles.css (hunk 3 above)
Comment (venturewise) [reply styles/styles.css:384]: This code was cropping all images, whether or not they were right-facing or left-facing. We have right/left clipping happening on other classes, I was going to look around for any images that are not addressed after this.
image
